scripts/ci/_utils.sh

```diff
@@ -1023,3 +1023,9 @@ function check_for_allowed_params {
     fi
     save_to_file "${_VARIABLE_NAME}"
 }
+
+
+function prepeare_backport_packages() {
+    ALL_PROVIDER_PACKAGES_
```

Comment:
Is it function or constant?

Reply:
Ah. That's a remainder from earlier test. will remove it.